src/db/migrations/28-add-mighty-33-others-die.ts

```diff
@@ -0,0 +1,54 @@
+import { Knex } from 'knex';
+
+import { IdExtractorTypes, TitleExtractorTypes } from '../../types/fieldExtractor';
+import { NftFactory, NFTContractTypeName, NFTStandard } from '../../types/nft';
+import { MusicPlatform, MusicPlatformType } from '../../types/platform';
+import { ethereumArtistId } from '../../utils/identifiers';
+import { addNftFactory, addPlatform, removeNftFactory, removePlatform } from '../migration-helpers';
+
+
+const mighty33Address = '0x8C62dD796e13aD389aD0bfDA44BB231D317Ef6C6';
+
+const MIGHTY_33: MusicPlatform = {
+  id: mighty33Address,
+  type: MusicPlatformType['multi-track-multiprint-contract'],
+  name: 'Mighty33',
+}
+
+const OTHERS_DIE: NftFactory = {
+  id: '0x97b9f21b41041e344f5bd71e3e86b69e79dcc0a6',
+  startingBlock: '10876005',
+  platformId: mighty33Address,
+  contractType: NFTContractTypeName.default,
+  standard: NFTStandard.ERC721,
+  autoApprove: true,
+  approved: true,
+  typeMetadata: {
+    overrides: {
+      artist: {
+        name: 'Mighty33',
+        artistId: ethereumArtistId(mighty33Address),
+        avatarUrl: 'https://web3-music-pipeline.mypinata.cloud/ipfs/QmS8dBxzgmNcCL71XYM5WxCkb2MM8rhQdCqAf1as8XXCVo',
+        websiteUrl: 'https://geniuscorp.fr/m33'
+      },
+      track: {
+        websiteUrl: 'https://geniuscorp.fr/m33'
+      },
+      type: MusicPlatformType['multi-track-multiprint-contract'],
+      extractor: {
+        title: TitleExtractorTypes.METADATA_NAME,
```

Comment:
i see the titles have track numbers in them like # 1, # 2 etc. i guess that's fine but may be nice to have an extractor that cuts those out, especially since it kinda looks bad with them in reverse album order

Reply:
in future we'll have support for album/re-ordering but for now just that is probably good